fastai/vision/learner.py

```diff
@@ -13,14 +13,22 @@
 def _default_split(m:nn.Module): return (m[1],)
 # Split a resnet style model
 def _resnet_split(m:nn.Module): return (m[0][6],m[1])
+# Returns body of sqeezenet style model
+def _squeezenet_body(m:nn.Module): return m.features
```

Comment:
I don't think this should be necessary any more.

Reply:
Agree, removed!